cv32e40x/tests/cfg/pma_test_cfg_5.yaml

```diff
@@ -2,9 +2,18 @@ name: pma_test_cfg_5
 description: PMA configuration for the PMA_TEST_CFG_5 test case
 compile_flags:
    +define+PMA_TEST_CFG_5
+   +define+ZBA_ZBB_ZBC_ZBS
 plusargs:
    +enable_pma=1
    +boot_addr=0x80
    +fix_sp=1
    +dm_halt_addr=0x301000
    +dm_exception_addr=0x301800
+   +enable_zba_extension=1
+   +enable_zbb_extension=1
+   +enable_zbc_extension=1
+   +enable_zbs_extension=1
+riscv_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
+gnu_march:   rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
+corev_march: rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
+llvm_march:  rv32imc_zba1p00_zbb1p00_zbc1p00_zbs1p00
```

Comment:
Does it exist any simple way to let e.g. PMA cfg yamls "include" B-extension cfg yamls? It seems like so many cfgs have have commonalities where it would be nice to just include this from a pre-defined "cfg module".

Reply:
We currently do not have any way to include this in multiple configurations - previously this was on a per-test basis.

Reply:
It would be "nice to have", to have a means for including re-usable chunks of configuration. Though what we have is simple enough and seems to be a good-enough solution. At least it is not per-test.